crates/bevy_audio/src/audio.rs

```diff
@@ -22,34 +21,12 @@ impl Default for Volume {
 
 impl Volume {
     /// Create a new volume level relative to the global volume.
-    pub fn new_relative(volume: f32) -> Self {
-        Self::Relative(VolumeLevel::new(volume))
+    pub fn new_relative(amplitude: f32) -> Self {
```

Comment:
It's been kept as-is to not introduce breaking changes, but I do think it would be better to have `new_relative` and `new_absolute` use the `VolumeLevel`. Or we can deprecate these ones and introduce new methods (meaning 4 methods total for the cartesian product of `(relative,absolute)x(amplitude,decibel)`?